pkgs/top-level/perl-packages.nix

```diff
@@ -1221,6 +1221,23 @@ let
     };
   };
 
+  BarcodeZBar = buildPerlPackage {
+    pname = "Barcode-ZBar";
+    version = "0.04";
+    src = "${pkgs.zbar.src}/perl";
```

Comment:
Can we use the CPAN source here for the perl package? Are there any drawbacks to doing that?